### ISSUE TITLE
Master doesn't test clean due to failure of loaded module

### ISSUE THREAD

**Issue by moxious** [graphaware/neo4j-nlp#22]
Full dump of a certain test run that fails:

https://gist.github.com/moxious/326ec7cba29bdc76975120998cbc1f33

As far as I can tell on tracing root causes, it's this:

```org.neo4j.graphdb.QueryExecutionException: Failed to invoke procedure `ga.nlp.ml.textRank`: Caused by: java.lang.RuntimeException: java.lang.NullPointerException```

`TextRankProcedureTest` extends `NLPIntegrationTest ` which in turn uses this setup:

```
        GraphAwareRuntime runtime = GraphAwareRuntimeFactory.createRuntime(getDatabase());
        runtime.registerModule(new NLPModule("NLP", NLPConfiguration.defaultConfiguration(), getDatabase()));
        runtime.start();
        runtime.waitUntilStarted();
        keyValueStore = new GraphKeyValueStore(getDatabase());
```

I'm guessing this is the mechanism used to start a neo4j instance with the module jacked in, but can't see further than this what the issue may be.  I traced it to here: https://graphaware.com/site/framework/latest/apidocs/com/graphaware/runtime/GraphAwareRuntime.html#registerModule-com.graphaware.runtime.module.RuntimeModule- but beyond that I'm uncertain.

**Comment by ikwattro** [comment]
Thanks @moxious . This should be fixed tomorrow. Sorry for this.

**Comment by ikwattro** [comment]
Fixed by #23
